scripts/validatePatches.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -e
+
+ROOT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+readonly ROOT_DIR
+readonly DETAILS_MD="details.md"
+ERRORS=()
+
+# Get list of modified patch files in PR (compared to main branch)
+CHANGED_PATCHES=$(git diff --diff-filter=AMR --name-only origin/main | grep "^patches/.*\.patch$" || true)
+REMOVED_PATCHES=$(git diff --diff-filter=D --name-only origin/main | grep "^patches/.*\.patch$" || true)
+
+if [ -z "$CHANGED_PATCHES" ] && [ -z "$REMOVED_PATCHES" ]; then
```

Comment:
from our (not yet public) bash style guide

```suggestion
if [[ -z "$CHANGED_PATCHES" ]] && [[ -z "$REMOVED_PATCHES" ]]; then
```

Reply:
Ah was not aware of this, is there some tracking for this or some WIP style guides?

Reply:
https://github.com/Expensify/Style-Guide/blob/main/shell.md#comparison-operations

Reply:
It would be possible to add a proper linter (for style, beyond just ShellCheck) for bash scripts. Just not sure if it's a priority

Reply:
Thanks for the link!